frontend/src/libraries/explorer-wamp/transactions.ts

```diff
@@ -137,7 +139,10 @@ export default class TransactionsApi extends ExplorerApi {
           ${whereClause.length > 0 ? `WHERE ${whereClause.join(" OR ")}` : ""}
           ORDER BY blocks.height ${filters.tail ? "DESC" : ""}
           LIMIT :limit`,
-        filters
+        {
+          filters,
+          limit
+        }
```

Comment:
This change looks completely broken to me. It is better to introduce `filters` object argument in the other Api methods, than moving limit out this way.

Reply:
@frol  filters are adding the `WHERE` clause to find the data.
limit is for `LIMIT :limit` to fetch the limit data.

<img width="1015" alt="image" src="https://user-images.githubusercontent.com/33007476/75507057-d01fd380-5994-11ea-91df-39c2f6f091de.png">
in above, limit is not the attribute for transaction table, but others are. if mixing them together, actually it is worse to make all api consistent and make sense to SQL query language.

Reply:
`filters` is an object which has/had various options passed to SQL query, e.g. `signerId`. With this change `:signerId` no longer a valid reference, as it is now nested into `filters`, so `:filters.signerId` (and I am not sure if that is even supported by sequelize). Thus, you should do `{ ...filters, limit }` to at least fix the functionality.

Given you still need to blend `limit` into the `filters`, I guess it is pointless to split them in the first place. The naming `filters` is not perfect; we can rename it to `query` or anything better if you have suggestions.